documentation/source/users/rmg/input.rst

```diff
@@ -771,6 +771,70 @@ to turn off pressure dependence for all molecules larger than the given number
+        globalAnalysis=False,
+        uncorrelated=True,
+        correlated=True,
+        localNumber=10,
+        globalNumber=5,
+        terminationTime=None,
+        pceRunTime=1800,
+        logx=True
+    )
+
+RMG can perform local uncertainty analysis using first-order sensitivity coefficients output by the native RMG solver.
+This is enabled by setting ``localAnalysis=True``. Performing local uncertainty analysis requires suitable settings in
+the reactor block (see :ref:`reactionsystem`). At minimum, the output species to perform sensitivity analysis on must
+be specified, via the ``sensitivity`` argument. RMG will then perform local uncertainty analysis on the same species.
+Species and reactions with the largest sensitivity indices will be reported in the log file and output figures.
+The number of parameters reported can be adjusted using ``localNumber``.
+
+RMG can also perform global uncertainty analysis, implemented using Cantera [Cantera]_ and the MIT Uncertainty
+Quantification (MUQ) [MUQ]_ library. This is enabled by setting ``globalAnalysis=True``. Note that local analysis
+is a required prerequisite of running the global analysis (at least for this semi-automatic approach), so
+``localAnalysis`` will be enabled regardless of the input file setting. The analysis is performed by allowing the
+input parameters with the largest sensitivity indices (as determined from the local uncertainty analysis) to vary
+while performing reactor simulations using Cantera. MUQ is used to fit a Polynomial Chaos Expansion to the resulting
+output surface. The number of input parameters chosen can be adjusted using ``globalNumber``.
+Note that this number applies independently to thermo and rate parameters and output species.
+For example ``globalNumber=5`` for analysis on a single output species will result in 10 parameters being varied, while
+having two output species could result in up to 20 parameters being varied, assuming no overlap in the sensitive input
+parameters for each output.
+
+The ``uncorrelated`` and ``correlated`` options refer to two approaches for uncertainty analysis. Uncorrelated means
+that all input parameters are considered to be independent, each with their own uncertainty bounds. Thus, the output
+uncertainty distribution is determined on the basis that every input parameter could vary within the full range of
+its uncertainty bounds. Correlated means that inherent relationships between parameters (such as rate rules for kinetics
+or group additivity values for thermochemistry) are accounted for, which reduces the uncertainty space of the input
+parameters.
+
+Finally, there are a few miscellaneous options for global uncertainty analysis. The ``terminationTime`` applies for the
+reactor simulation. It is only necessary if termination time is not specified in the reactor settings (i.e. only other
+termination criteria are used). The ``pceRunTime`` sets the time limit for fitting the PCE to the output surface.
```

Comment:
Define PCE?

Reply:
I added the abbreviation to the first mention of Polynomial Chaos Expansion earlier in the section (line 806).